src/plugins/multimedia/assets/mediacontrols.html

```diff
@@ -9,7 +9,8 @@
 		<div class="btn-group">
 			<button type="button" class="btn btn-default playpause" aria-controls="{{mId}}"  aria-labelledby="{{mId}}" title="{{play}}" data-state-on="{{play}}" data-state-off="{{pause}}"><span class="glyphicon glyphicon-play"><span class="wb-inv">{{play}}</span></span></button>
 			<button type="button" class="btn btn-default mute" aria-controls="{{mId}}" aria-labelledby="{{mId}}" title="{{mute_on}}" data-state-on="{{mute_on}}" data-state-off="{{mute_off}}" aria-pressed="false"><span class="glyphicon glyphicon-volume-up"><span class="wb-inv">{{mute_on}}</span></span></button>
-			<input type="range" class="volume" aria-controls="{{mId}}" aria-labelledby="{{mId}}" title="{{volume}}" min="0" max="100" value="100" step="5" />
+			<label id="{{mId}}-vlm-lbl" for="{{mId}}-vlm" hidden>{{volume}} 1</label>
```

Comment:
Because this `<label>` element is not used as being the main label for the input and it wont be. Please change it for a paragraph and keep it hidden.

Reply:
I had considered doing something along those lines, but in the end felt that a label might still be a better bet overall (even if it's not currently functioning like one in practice) since that's more in line with how that kind of relationship should traditionally be coded. But I guess you're right. It adds extra complexity for nothing. I'll change it.

IMO the way all of the media player controls are currently labelled is kind of strange. I find it inappropriate to be programmatically-identifying the video's title as every control's label. It's bloated/repetitive and relegates useful details about the controls to secondary information. At best it could maybe be argued that the video's title could be associated to the controls with ``aria-describedby`` (to relegate it to secondary information announced at the end), but even that might be a bit of a stretch.

Reply:
Updated - went with ``<span>`` instead of ``<p>``.

Reply:
`<span>` don't provide any semantic and `<span>` is a text level.